tests/scripts/deploy_admission_controller.sh

```diff
@@ -5,22 +5,52 @@ set -eo pipefail
 
 # Set our known directories and parameters.
 BASE_DIR=$(cd "$(dirname "$0")"; pwd)
+CERT_VERSION="v1.1.0"
```

Comment:
I see `v1.2.0` is out, why not use it instead?

Reply:
right, updated. thanks